src/main/java/org/folio/bulkops/client/InstanceClient.java

```diff
@@ -1,22 +1,16 @@
 package org.folio.bulkops.client;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import io.swagger.v3.oas.annotations.parameters.RequestBody;
 import org.folio.bulkops.domain.bean.BriefInstanceCollection;
-import org.folio.bulkops.domain.bean.Instance;
 import org.folio.bulkops.domain.bean.InstanceCollection;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 
 @FeignClient(name = "inventory/instances")
 public interface InstanceClient {
-  @PutMapping(value = "/{instanceId}")
```

Comment:
it's incorrect change, we should not update storage directly

Reply:
Fixed